torch/csrc/lazy/core/cache.h

```diff
@@ -63,6 +63,12 @@ class Cache {
     return it->second->second;
   }
 
+  TypePtr GetLatest() {
```

Comment:
is this needed? doesn't seem used

Reply:
Both this and the Numel methods was added when I inspect the state of the ComputationCache earlier. They are not used in this PR anymore. But should have a Numel method and GetLatest method for a LRU cache sounds natural?

Let me know if you want me to remove them